editor/scene_tree_editor.h

```diff
@@ -141,6 +141,8 @@ class SceneTreeEditor : public Control {
 	List<StringName> *script_types;
 	bool _is_script_type(const StringName &p_type) const;
 
+	int subscene_previously_clicked_at_time = 0;
```

Comment:
I think this won't compile on old compilers, move initialisation into constructor?